v2/openapi/fb-provider-off-exchange-api.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.0.0
+info:
+  title: 'Fireblocks Provider Off-Exchange API'
```

Comment:
We call an API a provider API, if it's something that a provider implements. In other cases it's just an API. Also we try to stick to the term "collateral" here, to avoid ambiguity. So, these things need to be changed:

1. The filename should be `fb-collateral-api.yaml`.
2. The title should be "Fireblocks Collateral API".
3. Add `info.description` key and explain there that this is an API implemented by Fireblocks as opposed to the other APIs here.